css/styles.css

```diff
@@ -479,52 +482,80 @@ footer a:hover {
 
     /*navbar*/
 
-    nav {
+    /*hide all links*/
+    .nav-links li, .search-box{
+        display: none;
+    }
+
+    .mobile-menu {
+        display: flex;
+        position: absolute;
+        right: 5%;
+        font-size: 22px;
+    }
+
+    /*he "responsive" class is added to the nav-links with JavaScript when the user clicks on the icon.*/
+
+    .nav-links.responsive li, .nav-links.responsive .search-box {
+      float: none;
+      display: block;
+      text-align: left;
+      padding: 20px 0px;
+    }
+
+    .navbar {
+        align-items: stretch;
         flex-direction: column;
+        gap: 2rem;
     }
-    .nav-links{
+
+    .logo-and-menu-btn-container {
+        display: flex;
+        align-items: center;
+    }
+
+    nav .responsive-nav {
         flex-direction: column;
     }
-    nav a, nav input[type=text] {
-        float: none;
+
+    nav .responsive li, nav .responsive input[type=text] {
         text-align: left;
         margin: auto;
 
     }
-    nav input[type=text] {
+
+    nav .responsive input[type=text] {
         border: 1px solid #ccc;
         width: 234px;
         height: 45px;
     }
 
```

Comment:
Is there any way we can add a transition effect to drop the menu down slowly? Maybe the same cubic bezier that I gave to the custom select?

Reply:
Good to remember it, I love  transition effect seems more interactive to the user. Done.

Reply:
The expand looks amazing!! Can we also get a collapsing effect too?